### ISSUE TITLE
Crash: Unhandled 'error' event

### ISSUE THREAD

**Issue by TheFrenchGhosty** [cloudrac3r/bibliogram#41]
```
      -> [OUT-NF ] https://www.instagram.com/graphql/query/?query_hash=[HASH]
events.js:293
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27)
Emitted 'error' event on ClientRequest instance at:
    at TLSSocket.socketErrorListener (_http_client.js:432:9)
    at TLSSocket.emit (events.js:316:20)
    at TLSSocket.EventEmitter.emit (domain.js:485:12)
    at emitErrorNT (internal/streams/destroy.js:84:8)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: -104,
  code: 'ECONNRESET',
  syscall: 'read'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bibliogram@1.0.0 start: `cd src/site && node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bibliogram@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/bibliogram/.npm/_logs/2020-03-17T18_21_30_241Z-debug.log
```

Content of ```/home/bibliogram/.npm/_logs/2020-03-17T18_21_30_241Z-debug.log``` 

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'start' ]
2 info using npm@[VERSION]
3 info using node@[VERSION]
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle bibliogram@1.0.0~prestart: bibliogram@1.0.0
6 info lifecycle bibliogram@1.0.0~start: bibliogram@1.0.0
7 verbose lifecycle bibliogram@1.0.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle bibliogram@1.0.0~start: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/bibliogram/bibliogram/node_modules/.>
9 verbose lifecycle bibliogram@1.0.0~start: CWD: /home/bibliogram/bibliogram
10 silly lifecycle bibliogram@1.0.0~start: Args: [ '-c', 'cd src/site && node server.js' ]
11 silly lifecycle bibliogram@1.0.0~start: Returned: code: 1  signal: null
12 info lifecycle bibliogram@1.0.0~start: Failed to exec start script
13 verbose stack Error: bibliogram@1.0.0 start: `cd src/site && node server.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:316:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:316:20)
13 verbose stack     at maybeClose (internal/child_process.js:1026:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid bibliogram@1.0.0
15 verbose cwd /home/bibliogram/bibliogram
16 verbose [KERNEL]
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "start"
18 verbose node [VERSION]
19 verbose npm  [VERSION]
20 error code ELIFECYCLE
21 error errno 1
22 error bibliogram@1.0.0 start: `cd src/site && node server.js`
22 error Exit status 1
23 error Failed at the bibliogram@1.0.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by cloudrac3r** [comment]
Probably fixed in `9fd9a00932d3d18783537e0395b41648e9fa0a9b`. Time will tell. If this does not happen again within the next week, it's likely that it's fixed.

**Comment by cloudrac3r** [comment]
Well, _my_ instance hasn't crashed yet, so I'm gonna call it fixed! Go me!
